soc/st/stm32/stm32f0x/Kconfig.defconfig

```diff
@@ -17,4 +17,14 @@ config TASK_WDT_HW_FALLBACK_DELAY
 	depends on TASK_WDT_HW_FALLBACK
 	default 100
 
+# Reduce kernel stack sizes to fit in 8 KiB RAM
+config MAIN_STACK_SIZE
+	default 640 if SOC_STM32F030X4 || SOC_STM32F030X8 || SOC_STM32F051X8
+
+config IDLE_STACK_SIZE
+	default 200 if SOC_STM32F030X4 || SOC_STM32F030X8 || SOC_STM32F051X8
+
+config ISR_STACK_SIZE
+	default 512 if SOC_STM32F030X4 || SOC_STM32F030X8 || SOC_STM32F051X8
```

Comment:
Let's work on a common Kconfig piece that would define profiles based on DT_CHOSEN(zephyr_ram) and apply it on the whole STM32 family.